src/components/ProductListItem.vue

```diff
@@ -5,7 +5,7 @@
     </ion-thumbnail>
     <ion-label>
       <h5>{{ item.brandName }}</h5>
-      <h2>{{ item.itemName }}</h2>
+      <h2>{{ getProduct(item.itemId).internalName }}</h2>
```

Comment:
I think we do not have to replace product name, we need to add internalName information. Please update